core/src/main/java/io/questdb/cairo/wal/ApplyWal2TableJob.java

```diff
@@ -116,6 +116,10 @@ public ApplyWal2TableJob(CairoEngine engine, int workerCount, int sharedWorkerCo
         mvStateWriter = new BlockFileWriter(config.getFilesFacade(), config.getCommitMode());
     }
 
+    public void applyTable(TableToken tableToken) {
```

Comment:
Return a boolean here that will show if the table is fully up to date.
Also, please add a test in this codebase that will use this method.